README.md

```diff
@@ -47,25 +51,31 @@ We are currently focused on a targeted subset of Python used for data-intensive
 2. *Non-Data Workloads:*
 Prioritize applications in data engineering, data science, and AI/ML. Bodo is not designed for general-purpose use cases that are non-data-centric.
 
```

Comment:
The non-goals section is focused on the compiler and not relevant to Bodo DataFrames except 2. What can we say here @strangeloopcanon @scott-routledge2 ?

Reply:
This section was mainly for the JIT compiler since it can be confusing, but dataframe library is a lot more clear so I think it's fine to remove it.